mono/metadata/remoting.c

```diff
@@ -2131,6 +2131,8 @@ mono_marshal_xdomain_copy_value_handle (MonoObjectHandle val, MonoError *error)
 	return result;
 }
 
+#ifndef DISABLE_REMOTING
```

Comment:
We should probably fix so that prototype and definition signatures of mono_marshal_xdomain_copy_value_handle are the same as well.

Reply:
I don't follow. You mean put `static` on the definition to match declaration? Yeah that'd be ok.

Reply:
Ok, I did that. Not sure it is what you meant.